src/docs/development/tools/android-studio.md

```diff
@@ -159,45 +161,82 @@ Also available in the "more actions" menu:
 * **Show Repaint Rainbow**: Shows rotating colors on layers when repainting.
 * **Hide Debug Mode Banner**: Hides the debug banner even when running a
   debug build.
-* **Auto horizontal scroll**: XXX - TBD
-* **Highlight nodes displayed in both trees**: XXX - TBD
+* **Highlight nodes displayed in both trees**: In the inspector, highlights
+  nodes shown in both the details and summary trees.
 * **Open Timeline View**: Analyzes activity of the application as it runs.
 * **Open Observatory**: Profiles Dart applications.
 
 ### Show performance data
 
 To view the performance data, including the widget rebuild information,
-start the app with **Debug**, and then open the Performance tool window
+start the app in **Debug** mode, and then open the Performance tool window
 using **View > Tool Windows > Flutter Performance**.
 
 ![Flutter performance window]({% asset tools/android-studio/widget-rebuild-info.png @path %})
 
 To see the stats about which widgets are being rebuilt, and how often,
 click **Show widget rebuild information** in the **Performance** pane.
-You can see how many times a widget was rebuilt in the last frame.
-For a high number of rebuilds, the exact count shows in the second column
-from the right, and a yellow spinning circle displays. 
-The column to the far right shows how many times a widget was rebuilt
-since entering the current screen. The app shown in this screenshot
-demonstrates poor performance.
-For widgets that weren't rebuilt, a solid grey circle displays.
+The exact count of the rebuilds for this frame displays in the second
+column from the right. For a high number of rebuilds, a yellow spinning
+circle displays. The column to the far right shows how many times a
+widget was rebuilt since entering the current screen.
+The app shown in this screenshot demonstrates poor performance.
+For widgets that aren't rebuilt, a solid grey circle displays.
 Otherwise, a grey spinning circle displays.
 
-### Debugging with Observatory
+The purpose of this tool is to make you aware when widgets are
+rebuilding&mdash;you might not realize that this is happening when just
+looking at the code. If widgets are rebuilding that you didn't expect,
+it's probably a sign that you should refactor your code by splitting
+up large build methods into multiple widgets.
+
+This tool can help you detect at least four common performance issues:
+
+1. The whole screen (or large pieces of it) are built by a single
```

Comment:
I probably just don't understand your markdown but why are these list items all `1.`?

Reply:
You can start each with a 1, but it will display correctly. Just a markdown thing.